wikiconv/.bazelrc

```diff
@@ -1,6 +1,5 @@
-tartup --host_jvm_args=-Xmx2500m
+startup --host_jvm_args=-Xmx2500m
 startup --host_jvm_args=-Xms2500m
```

Comment:
I'm a bit confused that you need exactly the same line twice in the `.bazelrc`, if it's needed, would be good to have a comment as to why.

Reply:
These are not the exact same line, Xms is the starting size, Xmx is the maximum size.
Arguably these parameters are unnecessary, but this is just correcting the typo at the beginning of the file.